homeassistant/components/synology_dsm/sensor.py

```diff
@@ -208,3 +215,37 @@ def device_info(self) -> Dict[str, any]:
             "sw_version": self._api.information.version_string,
             "via_device": (DOMAIN, self._api.information.serial),
         }
+
+
+class SynoNasSecuritySensor(SynoNasSensor):
+    """Representation a Synology Security sensor."""
+
+    @property
+    def state(self):
+        """Return the state."""
+        return getattr(self._api.security, self.sensor_type)
+
+    async def options_updated(self):
```

Comment:
I don't see where this is ever being called from.  It looks like it was copied from `unifi`

In `unifi`, this function is called via a dispatcher signal in `UniFiBase.async_added_to_hass`.  It doesn't look like one has been installed here.

`self.controller.signal_options_update, self.options_updated`

Reply:
Indeed we don't need that, nice review, as always :wink: